index.ts

```diff
@@ -148,14 +146,12 @@ export const onEvent: Plugin<CustomerIoPluginInput>['onEvent'] = async (event, m
 }
 
 async function syncCustomerMetadata(event: ProcessedPluginEvent, storage: StorageExtension): Promise<Customer> {
-    const customerStatusKey = `customer-status/${event.distinct_id}`
+    const customerStatusKey = `customer-status/${event.person.uuid}`
```

Comment:
would that not break existing users? 
If we want to do this we can, we might just want to roll this out as a separate v2 app

Reply:
True. If there's demand we could also write a script that will fetch the person uuid for every distinct_id inside of customer.io and merge it into that.